server/configs/application.properties

```diff
@@ -51,3 +51,9 @@ management.endpoints.web.exposure.include=*
 
 # Don't show the Spring banner on startup
 spring.main.banner-mode=off
+
+# Enable tomcat access log
+server.tomcat.basedir=@@pathToServer@@/build/deploy/embedded
```

Comment:
I'm not sure where this properties file is used. If it's used outside of dev machines, will this be a desirable path?

Reply:
It will not be desireable, in fact. I made a gradle plugin update to treat this like the other properties with `@@pathToServer@@`; to leave it commented out unless you're on a dev machine.

Reply:
Great